azurerm/internal/services/storage/storage_management_policy_resource.go

```diff
@@ -81,6 +84,35 @@ func resourceStorageManagementPolicy() *schema.Resource {
 										},
 										Set: schema.HashString,
 									},
+
+									"blob_index_match": {
+										Type:     schema.TypeSet,
+										Optional: true,
+										Set:      resourceStorageManagementPolicyBlobIndexMatchHash,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"tag_name": {
```

Comment:
then we can remove tag from these properties
```suggestion
												"name": {
```